routing_common/routing_common_tests/transit_test.cpp

```diff
@@ -0,0 +1,62 @@
+#include "testing/testing.hpp"
+
+#include "coding/reader.hpp"
+#include "coding/writer.hpp"
+
+#include "routing_common/transit_serdes.hpp"
+#include "routing_common/transit_types.hpp"
+
+#include <cstdint>
+#include <vector>
+
+namespace
+{
+using namespace routing;
+using namespace routing::transit;
+using namespace std;
+
+template <class Obj>
+void TestSerialization(Obj const & obj)
+{
+  vector<uint8_t> buffer;
+  MemWriter<vector<uint8_t>> writer(buffer);
+
+  Serializer<MemWriter<vector<uint8_t>>> serializer(writer);
+  obj.Visit(serializer);
+
+  MemReader reader(buffer.data(), buffer.size());
+  ReaderSource<MemReader> src(reader);
+  Obj deserializedObj;
+  Deserializer<ReaderSource<MemReader>> deserializer(src);
+  deserializedObj.Visit(deserializer);
+
+  TEST(obj.IsEqualForTesting(deserializedObj), ());
+}
+
+UNIT_TEST(ZeroTransitHeaderSerialization)
```

Comment:
I think that according to our default format it should be named "Transit_ZeroHeaderSerialization". (I'm not sure about the underscore.)
Here and in the other tests.

Reply:
Done